mozillians/users/migrations/0049_remove_unvouched.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.25 on 2020-09-28 14:25
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.contrib.auth.models import User
+
+
+def remove_unvouched(apps, schema_editor):
+    User.objects.filter(userprofile__is_vouched=False).delete()
```

Comment:
With the get_model, you can do something like
```
User = apps.get_model('auth', 'User')

Reply:
Will this remove also user profiles? Reversing this, why don't you delete UserProfile instead of User?

Reply:
> Reversing this, why don't you delete UserProfile instead of User?

I don't think that'll delete the User. But deleting the User does delete the profile as the profile has a OneToOne relation to it.

Reply:
The above will delete the User. What we want actually is to delete both the User and the Profile objects. During a data migration the signals will not work so you need to make sure that we are dropping both (maybe we do b/c of the cascade option)

Reply:
Yeah, seems like the default is to cascade when nothing is set: https://docs.djangoproject.com/en/1.11/_modules/django/db/models/fields/related/#OneToOneField. Will check the profiles are definitely deleted too.

One thing is that this is incredibly slow to perform, I guess because django is following the relation for every user to be deleted in turn. Happy to write this in SQL instead to be faster. (Although since we're only doing this once I'm not sure it's worth it)